cnames_active.js

```diff
@@ -1112,6 +1112,7 @@ var cnames_active = {
   "flamecord": "flamexode.github.io/flamecord",
   "flap": "slurmulon.github.io/flap",
   "flare": "nsaunders.github.io/flare",
+  "flaregram": "adityash4rma.github.io/flaregram-website"
```

Comment:
```suggestion
  "flaregram": "adityash4rma.github.io/flaregram-website",
```
As the CI error notes, your entry needs a comma at the end of it